src/components/SubTaskList/SubTaskList.jsx

```diff
@@ -7,57 +7,71 @@ import SubTaskAdd from './SubTaskAdd';
 import SubTaskProgressBar from './SubTaskProgressBar';
 
 export default class SubTaskList extends Component {
+  isComponentMounted = false;
```

Comment:
please operate with lifecycle methods, there's no need to check manually if component is mounted